build/__tools__/run-tests.sh

```diff
@@ -113,14 +114,20 @@ yell start tests with provisioning and enclave services
 say run unit tests for eservice database
 cd ${SRCDIR}/python/pdo/test
 try python servicedb.py --logfile $PDO_HOME/logs/client.log --loglevel info \
-    --eservice-db $PDO_HOME/data/db-test.json --url http://localhost:7101/ http://localhost:7102/ http://localhost:7103/ --ledger ${PDO_LEDGER_URL}
-try rm $PDO_HOME/data/db-test.json
+    --eservice-db ${ESDB_FILE} \
+    --url http://localhost:7101/ http://localhost:7102/ http://localhost:7103/ \
+    --ledger ${PDO_LEDGER_URL}
+try rm -f ${ESDB_FILE}
 
 cd ${SRCDIR}/build
 
 say create the eservice database using database CLI
-# add all enclaves listed in pcontract.toml
-try pdo-eservicedb --logfile $PDO_HOME/logs/client.log --loglevel info create
+try pdo-eservicedb --loglevel warn reset
+try pdo-eservicedb --loglevel warn add -u http://localhost:7101 -n es7101
+try pdo-eservicedb --loglevel warn add -u http://localhost:7102 -n es7102
+try pdo-eservicedb --loglevel warn add -u http://localhost:7103 -n es7103
+try pdo-eservicedb --loglevel warn add -u http://localhost:7104 -n es7104
+try pdo-eservicedb --loglevel warn add -u http://localhost:7105 -n es7105
 
```

Comment:
The create sequence of steps raises the following error : "failed to load database from file..". This seems to be an error generated unwantedly, haven't identified the part of  the logic that generates this unwanted error message. The database json file is indeed properly generated, and the rest of run-tests.sh works well as well. 

Also, interestingly the error message (even if by mistake) does not case run-tests.sh to fail. Looks like there are two problems here.

Reply:
create is implicit.

Reply:
and presumably your first point is what is referenced below

Reply:
done